deployments/liqo_chart/crds/config.liqo.io_clusterconfigs.yaml

```diff
@@ -6,9 +6,9 @@ metadata:
   annotations:
```

Comment:
I think we need a `make -f scripts/cluster-config/Makefile manifests ` to regenerate the crd, there is still v1